docs/nix-flakes.adoc

```diff
@@ -188,7 +188,7 @@ is similar to that of NixOS. The `flake.nix` would be:
 [source,nix]
```

Comment:
shouldn't we include or link templates/nix-darwin/flake.nix here ? else both flakes will go out of sync

Reply:
You mean, for example, like `include::../templates/nix-darwin/flake.nix[]`? That's actually a great idea.

Reply:
yes